### PR TITLE
Refactor to reflect did-io v1.0, crypto-ld and jsigs.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Refactor code to reflect changes in `did-io`, `crypto-ld`, and `jsonld-signatures`.
Addresses issue https://github.com/veres-one/did-veres-one/issues/54.

Depends on upstream PRs:
* https://github.com/veres-one/veres-one-context/issues/13
* ~~https://github.com/digitalbazaar/did-io/pull/50~~
* https://github.com/digitalbazaar/ocapld.js/pull/50
* ~~https://github.com/digitalbazaar/ed25519-signature-2018/pull/1~~
* ~~https://github.com/digitalbazaar/jws-linked-data-signature/pull/1~~
* ~~https://github.com/digitalbazaar/jsonld-signatures/pull/109~~

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
(Rebasing on latest `master`.)

**Comment by dmitrizagidulin** [comment]
(prepping for an npm release)

File: .eslintrc.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
 module.exports = {
   root: true,
-  extends: ['eslint-config-digitalbazaar'],
+  extends: [
+    'eslint-config-digitalbazaar',
+    // 'eslint-config-digitalbazaar/jsdoc'
```
Comment (gannan08) [inline .eslintrc.js:5]: Why is this disabled? This should be deleted entirely or re-enabled.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - name: Run test with Node.js ${{ matrix.node-version }}
+        run: npm run test-node
+#  test-karma:
```
Comment (gannan08) [inline .github/workflows/main.yml:37]: Are karma tests broken at the moment?

File: .eslintrc.js (hunk 1 above)
Comment (dmitrizagidulin) [reply .eslintrc.js:5]: Sure, I can remove it. (I added it in a disabled state mostly as a to-do item.)

File: .github/workflows/main.yml (hunk 2 above)
Comment (dmitrizagidulin) [reply .github/workflows/main.yml:37]: @gannan08 - yeah, they are. This lib didn't have a karma suite before; I started to add it in this PR, but it still needs more work.

File: .eslintrc.js (hunk 1 above)
Comment (dmitrizagidulin) [reply .eslintrc.js]: @gannan08 commented out line removed, opened a tracking issue at https://github.com/veres-one/did-veres-one/issues/59

**Comment by dmitrizagidulin** [comment]
Merging into the `v14.x` branch (not master), to make a `.beta` npm package release.

**Pull request merged**